go/vt/vttablet/tmrpctest/test_tm_rpc.go

```diff
@@ -45,7 +45,7 @@ import (
 // fakeRPCTM implements tabletmanager.RPCTM and fills in all
 // possible values in all APIs
 type fakeRPCTM struct {
-	t      *testing.T
+	t      testing.TB
```

Comment:
these changes exist to support the one lil' benchmark i wrote for the cached client